spec/features/audit/report/csv_export_spec.rb

```diff
@@ -8,25 +8,31 @@ def content_disposition
   end
 
   scenario "Exporting a csv file as an attachment" do
-    given_i_am_exporting_an_audit_report
-    then_i_receive_an_audit_progress_report_in_csv_format
+    given_i_am_an_auditor_belonging_to_an_organisation_with_audits
+    when_i_export_an_audit_report
+    then_i_receive_the_report_in_csv_format
   end
 
   scenario "Applying the filters to the export" do
-    given_i_have_applied_filters_and_exported_audits
+    given_i_am_an_auditor_belonging_to_an_organisation_with_audits
+    and_they_are_filtered
+    when_i_export_the_reports
     then_i_receive_an_audit_progress_report_for_filtered_audits_in_csv_format
   end
 
-  scenario "Multiple pagesincluding_details_of_audit_progress_ of content items are in the database" do
-    given_i_have_multiple_pages_of_content_items_and_do_not_filter_them
+  scenario "Multiple pages including_details_of_audit_progress_ of content items are in the database" do
+    given_i_am_an_auditor_belonging_to_an_organisation_with_audits
+    and_there_are_multiple_pages_of_unfiltered_content_items
+    when_i_export_all_the_reports
     then_i_receive_an_audit_progress_report_for_all_audits_in_csv_format
   end
 
   scenario "Discard audit status filter when clicking from content view to report, and then exporting CSV" do
-    given_i_apply_filters_to_the_audit_content_page
-    then_i_see_filtered_audits
-    given_that_i_navigate_to_audits_report_page
-    then_i_see_an_unfiltered_audit_progress_report
+    given_i_am_an_auditor_belonging_to_an_organisation_with_audits
+    when_i_apply_filters
```

Comment:
I still find this step a bit vague. What filters are you applying? What in your setup makes me expect that anything should be filtered by these filters (or not)? Maybe:

> Given I am an auditor belonging to an organisation
> And there are (unaudited, unallocated?) content items
> When I filter by (bla)
> Then I do (or don't) see them